GUI/QtQuick/qml/main.qml

```diff
@@ -138,7 +138,7 @@ ApplicationWindow {
         running: false
         repeat: false
         onTriggered: {
-            bsApp.getArmoryServers()
+            //bsApp.getArmoryServers()
```

Comment:
What's the purpose of this change? When will it setup the default connection instead?

Reply:
Bad commit...I have tried show wallets wizard without delay, but if I place function to other place also there is delay. So I dont like pr myself and I decline it)